workflows/speciesabundance.nf

```diff
@@ -129,13 +133,13 @@ workflow SpAnce {
     )
     ch_versions = ch_versions.mix(CSVTK_CONCAT.out.versions)
 
-    BRACKEN2KRONA (
+    KRAKEN2KRONA (
         KRAKEN2.out.report_txt
```

Comment:
The krona figure should be created from the numbers in the adjusted bracken report instead of directly from the kraken report so that the numbers in krona match what is being stored in the metadata in IRIDA Next.

Reply:
OK! All set! 
Changes can be found in [3a8113e](https://github.com/phac-nml/speciesabundance/pull/13/commits/3a8113eacc8a65af538d26cd9738f33f306d157e) AND [b9ce5e9](https://github.com/phac-nml/speciesabundance/pull/13/commits/b9ce5e9f572ae09b840fa2c63aafcb83d3014764) 😺 

Documentation updated in [9439b86](https://github.com/phac-nml/speciesabundance/pull/13/commits/9439b862c58d97eb955110f66834e0c8efec670e) 📃